synth.py

```diff
@@ -60,9 +63,17 @@
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(unit_cov_level=79, cov_level=79, samples_test=True)
+templated_files = common.py_library(cov_level=70)
+templated_files = common.py_library(cov_level=70, samples=True)
```

Comment:
Only the second line should be retained, or...?

Reply:
Done! Apologies for the delay.